src/glyph.rs

```diff
@@ -77,31 +87,43 @@ impl<R, F> CharacterCache for GlyphCache<R, F> where
                 }
             }
             Entry::Vacant(v) => {
-                self.face.set_pixel_sizes(0, size).unwrap();
-                self.face.load_char(ch as usize, ft::face::DEFAULT).unwrap();
-                let glyph = self.face.glyph().get_glyph().unwrap();
-                let bitmap_glyph = glyph.to_bitmap(RenderMode::Normal, None).unwrap();
-                let glyph_size = [glyph.advance_x(), glyph.advance_y()];
+                // fallback to glyph zero or U+FFFD if glyph is not present
+                let glyph = self.font.glyph(ch).unwrap_or(self.font.glyph(rt::Codepoint(0)).unwrap_or(self.font.glyph('\u{FFFD}').unwrap()));
```

Comment:
Just a small tidbit - I think this line could be refactored to use `or_else`, i.e.

```rust
let glyph = self.font.glyph(ch)
    .or_else(|| self.font.glyph(rt::Codepoint(0)))
    .or_else(|| self.font.glyph('\u{FFFD}')))
    .unwrap();
```

Using `or_else` will make the evaluation of `self.font.glyph(rt::Codepoint(0))` and `self.font.glyph('\u{FFFD}')` lazy and probably a little bit more efficient.

Similarly, lines 95 and 97 could probably use `unwrap_or_else(||` instead of `unwrap_or(`

Also, we should probably return some sort of `Err` instead of `unwrap`ing and panicking here.